torch/_inductor/codegen/cpp_micro_gemm.py

```diff
@@ -553,9 +566,17 @@ class CppMicroGemmAMX(CppMicroGemm):
         _tile_loadd({{tile_idx_a}}, A + {{tile_row * 16}} * lda + k, lda * sizeof({{input_t}}));
         {%- endif %}
         {%- if tile_row == 0 %}
+        {%- if int8_gemm %}
+        _tile_loadd({{tile_idx_b}}, B + k * ldb + {{tile_col * 16 * 4}}, ldb * 4 * sizeof({{input_t}}));
+        {%- else %}
```

Comment:
Can be simplified with "vnni_size"?

Reply:
Sure, added a render option of `vnni_size`